src/mcts/stoppers/factory.cc

```diff
@@ -42,9 +43,19 @@ const OptionId kMoveOverheadId{
     "Amount of time, in milliseconds, that the engine subtracts from it's "
     "total available time (to compensate for slow connection, interprocess "
     "communication, etc)."};
-const OptionId kTimeManagerId{"time-manager", "TimeManager",
-                              "Name and config of atime manager."};
-
+const OptionId kTimeManagerId{
+    "time-manager", "TimeManager",
+    "Name and config of atime manager. "
+    "Default is 'legacy'. "
+    "Optional time manager is 'alphazero' "
+    "'alphazero' Makes the engine use AlphaZero time. The alphazero engine always "
```

Comment:
It seems that the description gives disproportionally large attention to "alphazero" mode. :)
While leaving out all legacy parameters. I suggest leaving a link to `lczero.org/play/flags/timemgr/` (or `lc0.org/timemgr`) here, and creating a page on a website with the description.

Reply:
Moved explanation to the dev documentation, please see
https://github.com/LeelaChessZero/lczero.org/pull/29